controller/contacts.js

```diff
@@ -6,8 +6,40 @@ const {
 } = require("../joi");
 
 const get = async (req, res, next) => {
+  const { page, limit, favorite } = req.query;
+  let paginatedContacts = [];
+  const { _id } = req.user;
   try {
-    const contacts = await service.getAllContacts();
+    const contacts = await service.getAllContacts(_id);
+    if (!page && !limit && favorite) {
```

Comment:
Myślę, że paginację warto by było ograć ze strony bazy danych, dla niewielkich ilości danych to rozwiązanie zadziała, ale jednak Mongo poradzi sobie ze znacznie większymi ilościami
po metodzie find możesz podać metody `skip()` i `.limit()`, do filtrowania po favourite też warto użyć bazy i podać tę wartość w argumencie `find()`